erpnext/selling/page/point_of_sale/point_of_sale.js

```diff
@@ -120,13 +120,29 @@ erpnext.pos.PointOfSale = class PointOfSale {
 	}
 
 	make_cart() {
+		const me = this;
 		this.cart = new POSCart({
 			frm: this.frm,
 			wrapper: this.wrapper.find('.cart-container'),
 			events: {
 				on_customer_change: (customer) => {
 					this.frm.set_value('customer', customer);
 				},
+				on_order_type_change:(order_type) => {
+					me.frm.set_value('new_order_type', order_type)
+
+					if (order_type == __('Sample')) {
```

Comment:
Don't translate the string for validation, it'll always be "Sample" as an option